app/team_test.go

```diff
@@ -850,11 +850,11 @@ func TestGetTeamMembers(t *testing.T) {
 	sort.Sort(userIDs)
```

Comment:
Should we add tests to `TestGetTeamMembers` to verify the new behaviour?

Reply:
+1 definitely add new unit tests 
- test for the `API` to ensure proper parsing and accepting of the query parameters
- A few basic sanity checks on the `App` to ensure the sorting options pass through as intended
- More extensive tests at the store layer to ensure the queries are good.

Reply:
Added! Thanks